package.json

```diff
@@ -69,5 +69,12 @@
     "rollup-plugin-typescript": "^1.0.1",
     "tsd-jsdoc": "^2.1.3",
     "typescript": "^3.6.2"
+  },
+  "dependencies": {
+    "opencollective-postinstall": "^2.0.2"
+  },
+  "collective": {
+    "type": "opencollective",
+    "url": "https://opencollective.com/pixijs"
```

Comment:
I don't think we should do this.

Reply:
Also, this isn't the right place for this. This is a monorepo and there are 40+ packages. We already have a custom script in bundles/pixi.js that calls out opencollective. We could replace it with this script, potentially.